app/assets/javascripts/controllers/topology/container_topology_controller.js

```diff
@@ -13,49 +14,44 @@ function ContainerTopologyCtrl($scope, $http, $interval, topologyService, $windo
 
   topologyService.mixinContextMenu(vm, vm);
 
-  ManageIQ.angular.rxSubject.subscribe(function(event) {
-    if (event.name === 'refreshTopology') {
-      vm.refresh();
-    }
-  });
-
-  vm.refresh = function() {
```

Comment:
This `refresh` also dealt with URLs like `/container_project/show/11` leading to `/container_project_topology/11`.

Looks like we've lost that bit - Compute > Containers > Projects > ..project detail.. - Topology will be broken.

Reply:
nice catch, what about creating new property in controller `detailUrl` and store required path?

Reply:
Sure, that could work too :) (but will probably mean container topology does the same thing twice, so maybe just split that method in 2 and allow the controller to override the url bits?)